src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -115,6 +137,8 @@ export default {
 			searchResultsCircles: [],
 			contactsLoading: false,
 			isCirclesEnabled: loadState('talk', 'circles_enabled'),
+			attachmentFolder: loadState('talk', 'attachment_folder'),
```

Comment:
I guess in the end this needs to go into a store, otherwise the file upload from #2891 would still have the old value.

Reply:
@ma12-co where should we store this? New store? or can we put it in an existing one?